### PR TITLE
Ruslan/specialists

### PR DESCRIPTION
**Pull request by r-strilets**

### CODE REVIEW

File: src/components/ClientForms/BeAClientForm/BeAClientForm.jsx [hunk 1]
```diff
@@ -1,18 +1,18 @@
-import MessageSchema from "../MessageSchema";
-import Button from "@component/components/Button/Button";
-import { useFormik } from "formik";
-import { ErrorBox } from "./BeAClientForm.styled";
-import { postMessage } from "@component/utils/fetchAxiosData";
+import { MessageSchema } from '../../../utils/allValidate/MessageSchema';
+import Button from '@component/components/Button/Button';
+import { useFormik } from 'formik';
+import { ErrorBox } from './BeAClientForm.styled';
+import { postMessage } from '@component/utils/fetchAxiosData';
```
Comment (painlalala) [inline src/components/ClientForms/BeAClientForm/BeAClientForm.jsx:5]: Better to split imports
1.libs
free space
2. cunstom components

Comment (r-strilets) [reply src/components/ClientForms/BeAClientForm/BeAClientForm.jsx]: done

File: src/components/HomeParts/Specialists/Specialists.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                <LeftBox>
+                  <Image
+                    src={icon}
+                    alt={`${proffesion} icon`}
+                    style={{ width: '40px', height: '40px' }}
+                  />
+                  {proffesion}
+                </LeftBox>
+                <RightBox>{fullDesription}</RightBox>
+              </FullDescriptionBox> */}
```
[the remark this answers is not in the document]
Comment (r-strilets) [reply src/components/HomeParts/Specialists/Specialists.jsx:40]: it's code for window with animation, I commented it out so as not to break the main markup

File: src/components/ClientForms/BeAClientForm/BeAClientForm.jsx [hunk 3]
```diff
@@ -34,19 +42,28 @@ const BeAClientForm = ({ action }) => {
...
+        ></textarea>
+        {formik.touched.message && formik.errors.message ? (
+          <ErrorBox>{formik.errors.message}</ErrorBox>
+        ) : null}
       </label>
       <Button
-        title={"Відправити"}
-        type={"submit"}
+        title={'Відправити'}
+        type={'submit'}
```
[the remark this answers is not in the document]
Comment (r-strilets) [reply src/components/ClientForms/BeAClientForm/BeAClientForm.jsx:65]: in process

File: src/pages/index.js [hunk 4]
```diff
@@ -1,7 +1,9 @@
 // import { Section } from "@component/components/Section.styled";
 // import { Container } from "@component/components/Container.styled";
```
[the remark this answers is not in the document]
Comment (r-strilets) [reply src/pages/index.js]: done

**Review by painlalala — APPROVED** [review]

**Pull request merged**